### PR TITLE
Add snaphot range deletion to zfs plus quickemu and quickget

### PR DESCRIPTION
**Pull request by philclifford**

### CODE REVIEW

File: quickemu [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+tags: [ virtualization ]
+---
+# To run a vm
```
Comment (EmilySeville7cfg) [inline quickemu:4]:
```suggestion
# To run a specific vm
```

Optional change: `a specific` may sound better.

```suggestion
# To run a vm:
```

Optional change: trailing colon can be added.

```suggestion
# Run a vm
```

Optional change: `to` can be removed.

```suggestion
# Run a specific vm:
```

Optional change: fixes can be applied together.

File: quickemu [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+quickemu (--vm <path-to-vm.conf>)
```
Comment (EmilySeville7cfg) [inline quickemu:5]:
```suggestion
quickemu --vm <path-to-vm.conf>
```

Optional change: unexpected `(`: braces can be removed.

```suggestion
quickemu (--vm <path/to/vm.conf>)
```
Optional change: `path/to/...` format can be used.

```suggestion
quickemu --vm <path/to/vm.conf>
```

Optional change: fixes can be applied together.

File: quickemu [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+# To run a vm with a spice display
```
Comment (EmilySeville7cfg) [inline quickemu:7]:
```suggestion
# To run a specific vm with a spice display
```

```suggestion
# To run a vm with a spice display:
```

```suggestion
# Run a vm with a spice display
```

```suggestion
# Run a specific vm with a spice display:
```

The same issue as above.

File: quickemu [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+quickemu (--vm <path-to-vm.conf>) --display spice
```
Comment (EmilySeville7cfg) [inline quickemu:8]:
```suggestion
quickemu --vm <path-to-vm.conf> --display spice
```

```suggestion
quickemu (--vm <path/to/vm.conf>) --display spice
```

```suggestion
quickemu --vm <path/to/vm.conf> --display spice
```

The same issue as above.

File: quickemu [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+# To run a vm with a braille terminal display - Version > 3.11 and working brltty required
```
Comment (EmilySeville7cfg) [inline quickemu:10]:
```suggestion
# To run a specific vm with a braille terminal display - Version > 3.11 and working brltty required
```

```suggestion
# To run a vm with a braille terminal display - Version > 3.11 and working brltty required:
```

```suggestion
# Run a vm with a braille terminal display - Version > 3.11 and working brltty required
```

```suggestion
# Run a specific vm with a braille terminal display - Version > 3.11 and working brltty required:
```

The same issue as above.

File: quickemu [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+quickemu (--vm <path-to-vm.conf> --accessible braille --display sdl)
```
Comment (EmilySeville7cfg) [inline quickemu:11]:
```suggestion
quickemu --vm <path-to-vm.conf> --accessible braille --display sdl
```

```suggestion
quickemu (--vm <path/to/vm.conf> --accessible braille --display sdl)
```

```suggestion
quickemu --vm <path/to/vm.conf> --accessible braille --display sdl
```

The same issue as above.

File: quickemu [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+  #   or
```
Comment (EmilySeville7cfg) [inline quickemu:12]:
```suggestion
#   or
```

Optional change: indentation level can be decreased.

```suggestion
#   or:
```

The same issue as above.

```suggestion
# or:
```

Optional change: fixes can be applied together.

File: quickemu [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+quickemu (--vm <path-to-vm.conf> --acc brl --display sdl)
```
Comment (EmilySeville7cfg) [inline quickemu:13]:
```suggestion
quickemu --vm <path-to-vm.conf> --acc brl --display sdl
```

```suggestion
quickemu (--vm <path/to/vm.conf> --acc brl --display sdl)
```

```suggestion
quickemu --vm <path/to/vm.conf> --acc brl --display sdl
```

The same issue as above.

File: quickemu [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+
+## All the optional parameters
```
Comment (EmilySeville7cfg) [inline quickemu:15]:
```suggestion
## All the optional parameters:
```

The same issue as above.

**Review by EmilySeville7cfg — APPROVED** [review]
- No mandatory changes found, approved.
- `quickget`, `zfd` can be fixed to be consistent with `quickemu` page.
- Thanks for a contribution! ❤️

**Comment by philclifford** [comment]
> * Thanks for a contribution! 
Thanks for such helpful feedback. :heart: !

**Comment by philclifford** [comment]
I'll stop at that for `zfs` for now and re-visit the current (and pending) guidance here.
Thanks for a useful project!

**Comment by chrisallenlane** [comment]
This is great. Thanks!

**Pull request merged**